src/apps/Console/Console.js

```diff
@@ -316,10 +221,11 @@ function Prompt({ command, handleChange, handleSubmit }) {
               return
             }
             const nextHistory = clamp(
-              historyIndex + e.keyCode === KEYCODES.up ? -1 : 1,
+              historyIndex + (e.keyCode === KEYCODES.up ? -1 : 1),
               0,
               commandHistory.length - 1
```

Comment:
I was wondering; would `commandHistory` here allow us to clear the console if you pressed `down` past the first item in history (similar to most terminal history behaviour, so you can easily clear the input)?

Reply:
Yes; it would only involve checking if we're already clamped to the maximum value before and set the command to an empty string.